documentation/get-started/tutorials/get-started-qfc.en.md

```diff
@@ -56,12 +57,33 @@ A new login screen will appear:
 
 Enter your credentials previously created during account registration.
 
+!!! Note
+    If you use a password in QGIS for the first time, it will ask you to set a master password that manages all the other passwords used in QGIS. More information about the master password here: [QGIS documentation](https://docs.qgis.org/3.4/en/docs/user_manual/auth_system/auth_overview.html#master-password)
```

Comment:
```suggestion
    If you use a password in QGIS for the first time, it will ask you to set a master password that manages all the other passwords used in QGIS. More information about the master password here: [QGIS documentation](https://docs.qgis.org/latest/en/docs/user_manual/auth_system/auth_overview.html#master-password)
```